### ISSUE TITLE
`switch` should be followed by a space

### ISSUE THREAD

**Issue by hirasso** [PHP-CS-Fixer/PHP-CS-Fixer#7906]
## Bug report

I just found a minor issue: PHP CS Fixer doesn't insert a space after the `switch` keyword, so this won't be fixed:

```php
switch($value) {
    // ...
}
```

[PHP_CodeSniffer's](https://github.com/PHPCSStandards/PHP_CodeSniffer/) `phpcbf` fixes the missing space as expected:

```php
switch ($value) {
    // ...
}
```

### Description

Following [the relevant paragraph in PSR-2](https://www.php-fig.org/psr/psr-2/#52-switch-case), `switch` statements should be treated similarly to `if` statements and thus should be followed by a space.

### Runtime version

```
PHP CS Fixer 3.52.1 15 Keys by Fabien Potencier, Dariusz Ruminski and contributors.
PHP runtime: 8.2.0
```

### Used command

```
vendor/bin/php-cs-fixer fix --config .php-cs-fixer.config.php --allow-risky=yes
```

### Configuration file

```php
<?php

$finder = PhpCsFixer\Finder::create()
    ->in([
        'public/content/themes/site',
    ])
    ->name('*.php')
    ->notName('*.blade.php')
    ->ignoreDotFiles(true)
    ->ignoreVCS(true);

return (new PhpCsFixer\Config())
    ->setRules([
        '@PSR12' => true,
        '@PHP83Migration' => true,
        'no_unused_imports' => true,
    ])
    ->setFinder($finder);

```

### Code snippet that reproduces the problem

```php
switch($value) {
    // ...
}
```

**Comment by Wirone** [comment]
@hirasso thanks for the report. Please use `single_space_around_construct` to achieve what you need 🙂.

The interesting part is why it's not a part of PSR-2, it's only included in `@Symfony` (and inherited in `@PhpCsFixer`). @keradus @julienfalque do you have historical context?

**Comment by hirasso** [comment]
Thanks so much for the pointer towards `single_space_around_construct` @Wirone ! 

How would I activate it? By just setting it to `true` in my config?

**Comment by Wirone** [comment]
Well, it depends how you want it to work. As for any configurable rule you can use `true` for default config, or you can provide your own, using available options described in the docs.

**Comment by hirasso** [comment]
Thank you, also for this amazingly quick reaction time ⚡️🙏

**Comment by keradus** [comment]
some context about ruleset: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/4435#issuecomment-714363283

probably set could be groomed, but we cannot add all constructs to PSR ruleset.

**Comment by Wirone** [comment]
Should be fixed by #8171.
